src/odemis/gui/model/__init__.py

```diff
@@ -595,6 +595,40 @@ def __init__(self, main):
         # VA for autofocus procedure mode
         self.autofocus_active = BooleanVA(False)
 
+
+class LocalizationGUIData(MicroscopyGUIData):
+    # TODO: update the doc string
+    """ Represent an interface used to only show the current data from the microscope.
+
+    It should be able to handle SEM-only, optical-only, SECOM and DELPHI systems.
```

Comment:
Please do this TODO now. (And drop the `# TODO...`).
You can say it's for the CryoSECOM.